pypesto/visualize/parameters.py

```diff
@@ -404,8 +415,13 @@ def handle_inputs(
     if ub is None:
         ub = result.problem.ub_full
 
-    # get labels
-    x_labels = result.problem.x_names
+    # get labels as x_names and scales
+    x_labels = [
+        f"{x_name} ({x_scale})"
```

Comment:
If all parameters use the same scale, the scale could be included once in the x axis label instead of repeatedly in the tick labels.

Reply:
Hmmm true, will change

Reply:
I changed it, but it is quite a bit of changes to make it work, worth it?
Look at `x_axis_label` choice after `# If all the scales are the same, put it in the x_axis_label` comment at line 465, and then it has to be passed back and into `parameters_lowlevel`. Works tho, and the plot is nicer.

Reply:
Works for me, but it was an optional suggestion for now.



> and then it has to be passed back and into `parameters_lowlevel`.

Alternatively, you could override labels outside parameters_lowlevel, but this doesn't simplify things that much, right?

Reply:
Yeah not much, think it would be even uglier if it was outside of there

Reply:
Will see what the visualization ppl (@m-philipps and @stephanmg) think.

Reply:
Hi, looks fine to me, don't think the repetition of scale label is too bad. Certainly optional to think and implement a more visually pleasing appearance.